src/firefly/js/visualize/saga/CatalogWatcher.js

```diff
@@ -42,13 +42,15 @@ export function* watchCatalogs() {
 
 
     while (true) {
-        const action= yield take([TABLE_NEW_LOADED, TABLE_SORT, TABLE_SELECT,TABLE_HIGHLIGHT, TABLE_UPDATE,
+        const action= yield take([TABLE_LOADED, TABLE_SELECT,TABLE_HIGHLIGHT, TABLE_UPDATE,
                                   TABLE_REMOVE, ImagePlotCntlr.PLOT_IMAGE]);
         const {tbl_id}= action.payload;
         switch (action.type) {
-            case TABLE_SORT:
-            case TABLE_NEW_LOADED:
-                handleCatalogUpdate(tbl_id);
+            case TABLE_LOADED:
+                const {invokedBy} = action.payload;
+                if (invokedBy !== TABLE_SORT) {
```

Comment:
before, when sorting, the table highlight was the first row. So it might not be a bug.